bedrock/base/templates/includes/global-nav.html

```diff
@@ -211,6 +211,11 @@ <h3 class="summary" data-id="technology">
                   </a>
                 </li>
                 <li>
+                  <a href="{{ url('mozorg.developer') }}" data-link-type="nav" data-link-name="Mozilla Developers" data-link-group="Technology">
+                    {{ _('Mozilla Developers') }}
```

Comment:
Oh, I wasn't trying to suggest adding one to /technology I was just thinking of this new design hub page. Does that introduce string changes on /technology?